schematic/__init__.py

```diff
@@ -46,6 +58,59 @@
 USER_AGENT |= USER_AGENT_LIBRARY
 
 
+class FilterSensitiveData:
```

Comment:
I'm not sure that this makes sense to be a class since there isn't any state that needs to be tracked. These should probably be functions so you don't need to create an instance of the class like you had done here: `FilterSensitiveData().redacted_sensitive_data_in_exception(attributes)`

Reply:
Okay that makes sense. Thanks for the suggestion. I was initially thinking of making it a class so that I could group the functions together. But you are right since I don't need to track any states, a class might not be necessary. I think I can create a separate file called remove_sensitive_data_utils.py and save those functions there.